mozilla-release/browser/base/content/browser.js

```diff
@@ -858,9 +924,9 @@ function _loadURIWithFlags(browser, uri, params) {
         browser.webNavigation.setOriginAttributesBeforeLoading({ userContextId: params.userContextId });
       }
 
-      browser.webNavigation.loadURIWithOptions(uri, flags,
-          referrer, referrerPolicy, postData, null, null,
-          !!params.ensurePrivate);
+      browser.webNavigation.loadURIWithOptions(uri, flags, referrer, referrerPolicy,
```

Comment:
@alver-cliqz Please re-test if this breaks something. It also happens if we open about:preferences page.